### PR TITLE
#3 - Add a custom criterion and optimizer support for utils/classification.py 

### LINKED ISSUE #3: Add a custom criterion and optimizer support for utils/classification.py
Adding support for choosing custom loss functions and optimizers would allow users more flexibility while working out with their experiments.

You have to edit the `train_model()` function for this:

A solution for this issue would look somethinig like this:

`train_model(model=model, train_dataloader=train_dataloader, test_dataloader=test_dataloader, ...... optimizer=torch.optim.<Optimizer>(), criterion=torch.nn.<Criterion>)`

### LINKED ISSUE #4: Add a custom criterion and optimizer support for utils/segmentation.py
Adding support for choosing custom loss functions and optimizers would allow users more flexibility while working out with their experiments.

You have to edit the `train_model()` function for this:

A solution for this issue would look somethinig like this:

`train_model(model=model, train_dataloader=train_dataloader, test_dataloader=test_dataloader, ...... optimizer=torch.optim.<Optimizer>(), criterion=torch.nn.<Criterion>)`

### PR DESCRIPTION
**Pull request by AanshSamyani**

### CODE REVIEW

File: UniTrain/utils/classification.py [hunk 1]
```diff
@@ -1,7 +1,8 @@
 import os
+import UniTrain
 from torch.utils.data import DataLoader
 from torchvision import transforms
-from ..dataset.classification import ClassificationDataset
+from UniTrain.dataset.classification import ClassificationDataset
```
Comment (ishan121028) [inline UniTrain/utils/classification.py:5]: Please use the relative imports only.

**Review by ishan121028 — CHANGES_REQUESTED** [review]

**Comment by ishan121028** [comment]
Can you also modify the readme.md training example according to this?
